Backpack/Calendar/Classes/BPKCalendar.h

```diff
@@ -243,5 +243,10 @@ NS_SWIFT_NAME(Calendar) @interface BPKCalendar : UIView
  */
 - (void)reloadData;
 
+/**
+ * Refreshes the UI of every cell.
```

Comment:
This should specify that it's the visible cells right?

Reply:
Fixed